app/main.py

```diff
@@ -1 +1,26 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    command = command.split()
+    route = command[2].split("/")
+    if command[0] != "mv":
+        return
+    if len(route) > 1:
+        route.pop()
+    else:
+        new_file_name = route[0]
+        with (
+            open(command[1], "r") as file_in,
+            open(new_file_name, "x") as file_out,
+        ):
+            file_out.write(file_in.read())
+        os.remove(command[1])
+        return
+    for directory in route:
+        if not os.path.exists(directory):
+            os.mkdir(directory)
+        os.chdir(directory)
```

Comment:
Code Style: The checklist recommends using `os.path.join` for path concatenation. Here, you are changing the current working directory multiple times to navigate to the desired directory. This can lead to errors and make the code harder to understand. Instead, consider using `os.path.join` to build the path.